src/tests/home.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import '@testing-library/jest-dom/extend-expect';
+import Navbar from '../components/Navbar';
+
+describe('Nav Items', () => {
+  test('renders Home Navbar header', () => {
+    render(
+      <Router>
+        <Navbar />
+      </Router>,
+    );
+    const homeElement = screen.getByText('Home');
+    expect(homeElement).toBeInTheDocument();
+  });
+
```

Comment:
- [ ] You did a great job implementing the changes. :+1: While introducing new changes your tests broke. Kindly make sure all test cases are passing. 🤝️

![image](https://user-images.githubusercontent.com/17243325/234880789-5e11d989-9b04-43c5-b356-72901171f9ab.png)